visualization/app/codeCharta/ui/legendPanel/legendPanel.component.scss

```diff
@@ -70,19 +72,25 @@ cc-legend-panel {
 			.set-left-margin {
 				margin-left: 18px;
 			}
-
-			.marked-package {
-				overflow: hidden;
-				vertical-align: middle;
-				direction: rtl;
-				text-overflow: ellipsis;
-				max-width: 220px;
-				white-space: nowrap;
-				margin-top: -2px;
-			}
 		}
+
 		.bold {
 			font-weight: 600;
 		}
+
+		.cc-labelled-color-picker-container > .cc-labelled-color-picker-label {
+			white-space: nowrap;
+			text-overflow: ellipsis;
+			overflow: hidden;
+
+			/* This tricks the browser into cutting off the beginning instead of the end of the label. However,
```

Comment:
I am not too sure about the comments. They describe standard css. Maybe rename the css selector to `cc-cut-text-overflow-at-beginning` instead? On the other side there is no harm by the comments. I am fine with leaving it as it is as well.

Reply:
IMO this is still a misuse, or at least exploiting an edge case of standard CSS. The `direction: rtl;` property is not intended to switch text-overflow cutoff, but for right-to-left languages, which this is not. So as this a hack with some weird side effects, I want to make sure there is some warning at least.